From 7cb1d6e2d6c63e3bc4b275ff073152b550ba5b0f Mon Sep 17 00:00:00 2001
From: Jerome Glisse <jglisse@redhat.com>
Date: Mon, 16 Feb 2015 14:58:21 -0500
Subject: [PATCH] nouveau: fix unlock nouveau_bo_name_ref()
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Jérôme Glisse <jglisse@redhat.com>
---
 nouveau/nouveau.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nouveau/nouveau.c b/nouveau/nouveau.c
index da49e17..c6c153a 100644
--- a/nouveau/nouveau.c
+++ b/nouveau/nouveau.c
@@ -486,9 +486,9 @@ nouveau_bo_name_ref(struct nouveau_device *dev, uint32_t name,
 	if (ret == 0) {
 		ret = nouveau_bo_wrap_locked(dev, req.handle, pbo);
 		nouveau_bo((*pbo))->name = name;
-		pthread_mutex_unlock(&nvdev->lock);
 	}
 
+	pthread_mutex_unlock(&nvdev->lock);
 	return ret;
 }
 
-- 
1.8.3.1

